Jenkinsfile

```diff
@@ -13,13 +13,17 @@ configurationTypes = [
 
 cron_schedule = deployBranches.contains(BRANCH_NAME) ? '0 2 * * *' : ''
 
+def docker_registry = '979633842206.dkr.ecr.eu-west-1.amazonaws.com'
+def docker_registry_url = "https://${docker_registry}"
+def ecr_credential = 'ecr:eu-west-1:cita-devops'
+
 pipeline {
     triggers { cron(cron_schedule) }
     agent {
         label 'docker && awsaccess'
     }
     environment {
-        DOCKER_TAG = "${JOB_NAME}_${getSha()}"
+        DOCKER_TAG = "${env.BRANCH_NAME}_${getSha()}"
```

Comment:
we latch onto this for a **lot** of browserstack stuff. Can you give me some examples of how this will look:

See https://github.com/citizensadvice/design-system/blob/master/testing/features/support/helpers/env_variables.rb#L57-L61 and https://github.com/citizensadvice/design-system/blob/master/testing/features/support/drivers/browserstack/payload_values_generator.rb

Reply:
DOCKER_TAG looks like "PR-373_90ecb8f" or "master_90ecb8f" after these changes. I wouldn't have changed it if the "/" character in the job name didn't prevent docker-compose from connecting to the local docker host.

I could leave this and just change the CA_STYLEGUIDE_VERSION_TAG since that's the one that's being used.

Reply:
Although, looking at the files in you linked, I don't think anything's going to change in the browserstack reports if . Nothing's using the presence of "design-system/" in the DOCKER_TAG variable, and delete_prefix just returns the original string if the prefix isn't there.

Reply:
So previously it was `design_system/master_abcdef` or `design_system/PR-400_abcdef` and now it's going to be `master_abcdef` or `PR-400_abcdef`?

Reply:
That's it. Unless it causes an issue somewhere.